fullscreen.bs

```diff
@@ -182,6 +182,12 @@ if all of the following are true, and false otherwise:
 <ul>
  <li><p><var>element</var> is <a>connected</a>.
 
+ <li><p><var>element</var>'s <a>node document</a>'s <a>fullscreen element</a> is null, or is an
+ <a>inclusive ancestor</a> of <var>element</var>.
```

Comment:
Oh, right, the ready check used to be recursive but I removed that in https://github.com/whatwg/fullscreen/pull/52.